src/containers/orders/cart/filled-cart/filled-cart.js

```diff
@@ -33,7 +33,7 @@ const FilledCart = ({ items }) => {
     })
   );
 
-  const isLightTheme = useContext(ThemeContext);
+  const [isLightTheme] = useContext(ThemeContext);
```

Comment:
tests have fallen because of using an array to destructure. You can fix it by mocking useContext, smth like jest.mock('react', () => ({
  ...jest.requireActual('react'),
  useContext: () => [true, () => null]
}));
The same with other components.

Reply:
fixed